custom_components/bambu_lab/pybambu/models.py

```diff
@@ -1191,6 +1191,7 @@ class PrintErrorList:
     _count: int
 
     def __init__(self, client):
+        self._error = None
         self._client = client
         self._error = {}
```

Comment:
should that still stay?

Reply:
Good catch. No, that should be removed although it's probably benign as in practically all cases it'll be overwritten by None on the very first printer update.